android/utility/i18n-check-unused-strings.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+#
+# This script finds all references to strings in strings.xml in order to make sure that old
```

Comment:
Can we note the similarity to the Windows script? (I don't think it's worth merging them at this time)